docs/content/latest/troubleshoot/cluster/replace_tserver.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Replace failed tserver
+linkTitle: Replace failed tserver
+description: Replace failed tserver
+aliases:
+  - /troubleshoot/cluster/reaplace-tserver/
+  - /latest/troubleshoot/cluster/replace-tserver/
+menu:
+  latest:
+    parent: troubleshoot-cluster
+    weight: 829
+isTocNested: true
+showAsideToc: true
+---
+
+Suppose you have a cluster running and now a tserver has failed and needs to be replaced. The procedure to follow in this 
+scenario is:
+
+## Start new yb-tserver
+
+Start up a new yb-tserver, make sure it is in the same placement group as the one you are replacing!
+
+Refer to [starting `tserver` servers](../../../../deploy/manual-deployment/start-tservers/) for further parameters and options.
+
+## Blacklist the old yb-tserver
+
+Blacklist the old yb-tserver: 
+```bash
+~/master/bin/yb-admin -master_addresses $MASTERS change_blacklist ADD $OLD_IP:9100
+```
+
+Refer to [change_blacklist](../../admin/yb-admin.md#change-blacklist) command for further parameters and options.
+
```

Comment:
```suggestion
## Blacklist the failed YB-TServer

To blacklist the failed YB-TServer, run the following command:
 
```sh
~/master/bin/yb-admin -master_addresses $MASTERS change_blacklist ADD $OLD_IP:9100
```

For details, see [`change_blacklist`](../../admin/yb-admin.md#change-blacklist) command in the `yb-admin` reference page.

```

Reply:
Add an explanation about why you need to blacklist this address.

Reply:
@stevebang it's explained on the next step that the master is draining the failed yb-tserver

Reply:
I've committed all requested changes.